docs/DeveloperGuide.adoc

```diff
@@ -361,22 +367,35 @@ Windows, Linux, Unix, OS-X
 [[private-contact-detail]] Private contact detail::
 A contact detail that is not meant to be shared with others
 
+[[logging]] Logging::
+Logging is the use of logging tools to track the performance of the application while it is running.
+
+[[participant]] Participant::
+It represents a participant taking part in the hackathon
+
+[[mentor]] Mentor::
+It represents a mentor available for teams to choose
+
+[[team]] Team::
+Team is the base unit of this project. It contains references to an associated list of participants and an optional mentor. 
+
```

Comment:
I'm not really sure, as this is on explaining UI components, do we have to explain 'TextBox', 'MessageBox', 'EntityCard'(as it was defined in the Glossary of some applications.

Reply:
I think we can update later once it is in